agave_app/FilmWidget.cpp

```diff
@@ -1,181 +0,0 @@
-  setLayout(&m_Layout);
-
-  // Exposure, controls how bright or dim overall scene is
-  m_ExposureSlider.setStatusTip(tr("Set Exposure"));
-  m_ExposureSlider.setToolTip(tr("Set camera exposure"));
-  m_ExposureSlider.setRange(0.0f, 1.0f);
-  m_ExposureSlider.setValue(cam->GetFilm().GetExposure());
-  m_ExposureSlider.setDecimals(3);
-  m_ExposureSlider.setSingleStep(0.01);
-
-  m_Layout.addRow("Exposure", &m_ExposureSlider);
-
-  QObject::connect(&m_ExposureSlider, SIGNAL(valueChanged(double)), this, SLOT(SetExposure(double)));
-
-  // Number of render iterations per viewport update
-  m_ExposureIterationsSpinner.setStatusTip(tr("Set Exposure Time"));
-  m_ExposureIterationsSpinner.setToolTip(tr("Set number of samples to accumulate per viewport update"));
-  m_ExposureIterationsSpinner.addItem("1", 1);
-  m_ExposureIterationsSpinner.addItem("2", 2);
-  m_ExposureIterationsSpinner.addItem("4", 4);
-  m_ExposureIterationsSpinner.addItem("8", 8);
-  m_ExposureIterationsSpinner.setCurrentIndex(
-    m_ExposureIterationsSpinner.findData(cam->GetFilm().GetExposureIterations()));
-  m_Layout.addRow("Exposure Time", &m_ExposureIterationsSpinner);
-  QObject::connect(&m_ExposureIterationsSpinner,
-                   SIGNAL(currentIndexChanged(const QString&)),
-                   this,
-                   SLOT(SetExposureIterations(const QString&)));
-
-  m_NoiseReduction.setStatusTip(tr("Enable denoising pass"));
-  m_NoiseReduction.setToolTip(tr("Enable denoising pass"));
-  m_NoiseReduction.setCheckState(rs->m_DenoiseParams.m_Enabled ? Qt::CheckState::Checked : Qt::CheckState::Unchecked);
-  m_Layout.addRow("Noise Reduction", &m_NoiseReduction);
-
-  QObject::connect(&m_NoiseReduction, SIGNAL(stateChanged(const int&)), this, SLOT(OnNoiseReduction(const int&)));
-
-  QObject::connect(&cam->GetFilm(), SIGNAL(Changed(const QFilm&)), this, SLOT(OnFilmChanged(const QFilm&)));
-}
-
-void
-QFilmWidget::SetPresetType(const QString& PresetType)
-{
-  if (PresetType == "NTSC D-1 (video)") {
-    m_Preset[0].SetPreset(720, 486);
-    m_Preset[1].SetPreset(200, 135);
-    m_Preset[2].SetPreset(360, 243);
-    m_Preset[3].SetPreset(512, 346);
-  }
-
-  if (PresetType == "NTSC DV (video)") {
-    m_Preset[0].SetPreset(720, 480);
-    m_Preset[1].SetPreset(300, 200);
-    m_Preset[2].SetPreset(360, 240);
-    m_Preset[3].SetPreset(512, 341);
-  }
-
-  if (PresetType == "PAL (video)") {
-    m_Preset[0].SetPreset(768, 576);
-    m_Preset[1].SetPreset(180, 135);
-    m_Preset[2].SetPreset(240, 180);
-    m_Preset[3].SetPreset(480, 360);
-  }
-
-  if (PresetType == "PAL D-1 (video)") {
-    m_Preset[0].SetPreset(720, 576);
-    m_Preset[1].SetPreset(180, 144);
-    m_Preset[2].SetPreset(240, 192);
-    m_Preset[3].SetPreset(480, 384);
-  }
-
-  if (PresetType == "HDTV (video)") {
-    m_Preset[0].SetPreset(1920, 1080);
-    m_Preset[1].SetPreset(490, 270);
-    m_Preset[2].SetPreset(1280, 720);
-    m_Preset[3].SetPreset(320, 180);
-  }
-}

```

Comment:
Just checking, is there still a way to change these presets? Were they removed because this all got moved to the Render dialog?

Reply:
Yeah this has been dead code since a long time ago. It is some last vestige from pre-1.0 agave.  The only render resolution settings are now in the Render Dialog.  The main viewport just always uses the window size, for now.